services/common/src/constants/forms.js

```diff
@@ -0,0 +1,20 @@
+// Redux Forms
+export const ADD_REPORT = "ADD_REPORT";
```

Comment:
I think I also remade forms.js as a forms enum. Still works pretty much the same way but it only has one entry so far and doesn't trigger duplicate named import warnings/errors.